storage/sql/accounts/get.sql

```diff
@@ -28,7 +28,20 @@ SELECT
 	"rank",
 	"fees",
 	"rewards",
-	"vote"
+	"vote",
+	"u_username",
+	"u_isDelegate"::int::boolean,
+	"u_secondSignature"::int::boolean,
+	"u_nameexist"::int::boolean as "u_nameExist",
+	"u_multimin" as "u_multiMin",
+	"u_multilifetime" as "u_multiLifetime",
+	"u_balance",
+	case
+	when
+		"producedBlocks" + "missedBlocks" = 0 then 0
+	else
+		(("producedBlocks"::float / ("producedBlocks" + "missedBlocks")) * 100.0)::integer
```

Comment:
Then the logic of logic/accounts.js, `calculateProductivity` function is redundant.

Reply:
`calculateProductivity` was removed as part of 1.5 development